ee/packages/federation-matrix/src/events/helpers.ts

```diff
@@ -0,0 +1,103 @@
+import { Room } from '@rocket.chat/core-services';
```

Comment:
a file named helpers usually means that we have no real/clear idea why those functions were created and used